edge-patron/src/main/resources/domain/edge-patron/features/samples/fine/create-fee-entity-request.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please format all of the JSON files consistently. Maybe a JSON formatter e.g. https://jsonformatter.curiousconcept.com/# or a command line tool could be used to achieve this.